airflow/bin/cli.py

```diff
@@ -1529,6 +1576,10 @@ class CLIFactory(object):
             'func': pool,
             'help': "CRUD operations on pools",
             "args": ('pool_set', 'pool_get', 'pool_delete'),
+        }, {
+            'func': pools,
+            'help': "CRUD operations on a batch of pools",
+            "args": ('airflow_pools_yaml_filepath',),
```

Comment:
If you could somewhere present a sample of what this YAML looks like, it would be very helpful

It can be hard to intuit config file formats from reading code.

One good way to give a sample YAML file would be writing a unit test.

Reply:
Do you mean test in etl repo or test in incubator-airflow? We don't have anywhere running incubator-airflow test, right?

Reply:
I'm not sure. I would like them in this directory, because they're testing airflow functionality. Maybe we hook some CI up to airflow?

But yeah, putting them in etl will also be sufficient.